polar-core/src/rules.rs

```diff
@@ -18,66 +16,17 @@ impl Parameter {
     }
 }
 
-#[derive(Debug, Clone, Deserialize, Serialize)]
+#[derive(Debug, Clone, Deserialize, PartialEq, Serialize)]
 pub struct Rule {
     pub name: Symbol,
     pub params: Vec<Parameter>,
     pub body: Term,
-    #[serde(skip, default = "SourceInfo::ffi")]
-    pub source_info: SourceInfo,
```

Comment:
Why are you getting rid of source info? Don't we need that to return good errors?

Reply:
I'm only getting rid of it from rules — not from terms. We don't currently use it anywhere for rules.

Reply:
It seems to me like we'll probably want to pretty soon, no? I don't know why we'd make that harder on ourselves by undoing the work to enable it

Reply:
Maybe. It's just as easy to bring it back when we need it instead of having it hang around now, unused, but I don't really care either way. I will add it back.